client/lib/plans/constants.js

```diff
@@ -0,0 +1,22 @@
+export const plansList = {
+	'free-plan': {},
+	'premium-plan': {},
+	'business-bundle': {}
+};
+
+export const featuresList = {
+	'google-analytics': {
+		title: 'Google Analytics Integration',
```

Comment:
If we do that, then we'll have either the same name copied all over OR someone will phrase it differently in other places.